backend/apigateway/src/main/resources/application.properties

```diff
@@ -1,60 +1,6 @@
-## Profile specific properties
-spring.profiles.active=@activatedProperties@
-
-## maven directory
+management.endpoints.web.exposure.include=*
```

Comment:
It would make sense that this is a new file and the previous is actually kept in the monolith